task08/task08/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task08
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите x");
+            double x = int.Parse(Console.ReadLine());
+            FunctionEvaluation(ref x);
+            Console.WriteLine("F(x) = " + FunctionEvaluation(ref x));
+            Console.ReadKey();
+        }
+    
+        static double FunctionEvaluation(ref double x)
```

Comment:
1. Используйте правильный тип для интерпретации ввода.
2. Лишний вызов метода.
3. Не используйте без особых причин передачей аргумента в метод по ссылке.
```suggestion
            double x = double.Parse(Console.ReadLine());
            Console.WriteLine("F(x) = " + FunctionEvaluation(x));
            
            Console.ReadKey();
        }
    
        static double FunctionEvaluation(double x)
```